RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -1961,9 +1797,83 @@ boolean auto_deleteMail(kmailObject msg)
 	return false;
 }
 
+boolean LX_summonMonster()
+{
+	// get war outfit if have yr available
+	if(!possessOutfit("Frat Warrior Fatigues") && auto_warSide() == "fratboy" && adjustForYellowRayIfPossible())
+	{
+		// attempt to use calculate the universe
+		if(summonMonster($monster[War Frat 151st Infantryman])) return true;
+		// attempt to summon other sources of outfit
+		if(summonMonster($monster[War Frat Mobile Grill Unit])) return true;
+		if(summonMonster($monster[orcish frat boy spy])) return true;
+	}
+	if(!possessOutfit("War Hippy Fatigues") && auto_warSide() == "hippy" && adjustForYellowRayIfPossible())
+	{
+		if(summonMonster($monster[War Hippy Airborne Commander])) return true;
+		if(summonMonster($monster[war hippy spy])) return true;
+	}
+
+	// summon mountain man if we know the ore we need and still need 2 or more
+	item oreGoal = to_item(get_property("trapperOre"));
+	if(internalQuestStatus("questL08Trapper") < 2 && oreGoal != $item[none] && item_amount(oreGoal) < 2 && adjustForYellowRayIfPossible())
+	{
+		if(summonMonster($monster[mountain man])) return true;
```

Comment:
Agree it could happen but imo risk is low enough to not write extra code for it. Any insta kill yellow ray will get used prior to envy. If a user says it happens to them, then we can work on it